src/personio_py/client.py

```diff
@@ -127,8 +127,7 @@ def request_json(self, path: str, method='GET', params: Dict[str, Any] = None,
                during this request (default: True for json requests)
         :return: the parsed json response, when the request was successful, or a PersonioApiError
         """
-        headers = {'accept': 'application/json'}
-        response = self.request(path, method, params, data, headers, auth_rotation=auth_rotation)
+        response = self.request(path, method, params, data, auth_rotation=auth_rotation)
```

Comment:
was there a reason to remove the headers from the request?

Reply:
setting json= .. instead of data=.. in the request automatically sets the accept header to 'application/json'